docs/schema/guidance/repr-trusts.rst

```diff
@@ -0,0 +1,101 @@
+
+.. figure:: ../../_assets/trust-scenario1.svg
+   :alt: Diagram show linked statements. Entity statement labeled Trust A with entityType 'arrangement' and entitySubtype generalCategory 'trust.' Trust A is connected to person A by a relationship statement with interest type 'settlor', person B by a relationship statement with interest type 'trustee', and person C by a relationship statement with interest type 'beneficiary'
+   :figwidth: 80%
+   :align: center
+   
+Scenario 2
+^^^^^^^^^^
+A trust with a known beneficiary, known trustee, known settlor and known protector. The trust has a legal personality. 
+
+.. figure:: ../../_assets/trust-scenario2.svg
+   :alt: Diagram show linked statements. Entity statement labeled Trust A with entityType 'legalEntity' and entitySubtype generalCategory 'trust.'  Trust A is connected to person A by a relationship statement with interest type 'settlor', person B by a relationship statement with interest type 'trustee', person C by a relationship statement with interest type 'protector', and person D by a relationship statement with interest type 'beneficiary.' 
+   :figwidth: 100%
+   :align: center
+   
+   
+Scenario 3
+^^^^^^^^^^
+A trust where the settlor is also a trustee and the beneficiary is anonymous due to a disclosure exemption.
+
+.. figure:: ../../_assets/trust-scenario3.svg
+   :alt: Diagram show linked statements. Trust A is connected to person A by a relationship statement with interest types 'settlor' and 'trustee', person B by a relationship statement with interest type 'trustee', person C by a relationship statement with interest types 'beneficiary.' Person D has a person statement with person type 'anonymous person' and unspecified person details 'subject exempt from disclosure'
+   :figwidth: 95%
+   :align: center
+
+Scenario 4
+^^^^^^^^^^
+<<<<<<< HEAD
+A trust-like arrangement where local names are provided for the type of entity and the interests. 
+=======
+A trust-like arrangement with local names for the entityType and interestTypes. 
+>>>>>>> 3ebd0b3 (updating trusts reference page after review)
+
+.. figure:: ../../_assets/trust-scenario4.svg
+   :alt: Diagram show linked statements. Entity statement labeled Trust A with entityType 'arrangement', entitySubtype generalCategory 'trust', and entitySubtype localTerm 'fiducie-sûreté.' Trust A is connected to person A by a relationship statement with interest type 'settlor' and interest details 'constituant', person B by a relationship statement with interest type 'trustee' and interest details 'fiduciaire', and person C by a relationship statement with interest type 'beneficiary' and interest details 'beneficiaire'
+   :figwidth: 82%
+   :align: center
+
+Scenario 5
+^^^^^^^^^^
+A significant stake in a company is held in a trust. The parties to the trust are the beneficial owners of the company. Dotted lines represent the further Relationship Statements needed to represent the indirect beneficial ownership of the parties. 
```

Comment:
Should this be 'Relationship statements' - ie. uncapitalised 'S'?

Reply:
In the style guide we currently have 'Schema objects should be referred to by their title, (e.g. 'the Interested Party object has a number of required properties', 'The details of the company are entered into an Entity Statement')' but could change that if it reads weird?

Reply:
No - that's fine. It's in the style guide that property titles should be fully capitalised, I'd just forgotten.